xbmc/settings/windows/GUIWindowSettingsScreenCalibration.cpp

```diff
@@ -178,15 +217,19 @@ bool CGUIWindowSettingsScreenCalibration::OnMessage(CGUIMessage& message)
     break;
   case GUI_MSG_CLICKED:
     {
-      // clicked - change the control...
+      // On click event select the next control
       NextControl();
     }
     break;
   case GUI_MSG_NOTIFY_ALL:
     {
-      if (message.GetParam1() == GUI_MSG_WINDOW_RESIZE)
+      if (message.GetParam1() == GUI_MSG_WINDOW_RESIZE &&
+          message.GetSenderId() != WINDOW_SCREEN_CALIBRATION && IsActive())
       {
+        m_Res.clear();
+        CServiceBroker::GetWinSystem()->GetGfxContext().GetAllowedResolutions(m_Res);
```

Comment:
This is something we should probably try to get rid of later. It's not something you have introduced with this changes (so this is not a change request nor will it block the PR).

It does not make much sense to be calling upwards to get the context or ask for allowed/or current resolutions. Ideally this should be propagated to this component somehow. Dialogs/windows need to react to resolution changes in some event driven way (informed by the parent)